### DISCUSSION TITLE
Big Data Related Tools - Hadoop

### DISCUSSION THREAD

**Discussion by Abidox808** [community/community#122787] [Programming Help]
### Body

What do you find most interesting about HDFS, and how does it enable fault tolerance and scalability in Hadoop clusters?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Marouane-Elgoumiri** [comment]
HDFS's design incorporates fault tolerance and scalability, crucial for managing big data applications:

Data in HDFS is split into blocks (typically 128 MB in size), and each block is replicated across multiple nodes in the cluster (default is three replicas). and scalability in HDFS is facilitated by its distributed architecture. As data grows, more nodes can be added to the cluster seamlessly, which allows the file system to scale horizontally to handle more data.
